src/layout/Header/Header.tsx

```diff
@@ -1,57 +1,63 @@
 import { useTranslation } from 'react-i18next';
-// import { LANGUAGES } from '../../config/constants';
-import AppBar from '@mui/material/AppBar';
-import Box from '@mui/material/Box';
-import { StyledToolBar } from './Header.styled';
-import IconButton from '@mui/material/IconButton';
+import { useState } from 'react';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { toggleTheme } from '../../app/features/themeSlice';
+import { logout, showAuthModal } from '../../app/features/authSlice';
 import {
-    Avatar,
+    Box,
+    AppBar,
     Button,
     Link,
     Menu,
     MenuItem,
     Typography,
+    IconButton,
+    ListItemIcon,
 } from '@mui/material';
+import { NavLink, useNavigate } from 'react-router-dom';
+import { StyledButton } from '../Footer/Footer.styled';
```

Comment:
You could add a seperate styledButton for header instead of using from footer component. Or use a shared button.

Reply:
Updated.